src/app/pages/SearchPage/searchPage.tsx

```diff
@@ -0,0 +1,36 @@
+import Search from "../../components/Search/search";
+import styles from "./searchPage.module.css";
+
+export default function SearchPage() {
+  return (
+    <div className={styles.container}>
+      <Search />
+      <footer className={styles.footer}>
+        <svg
+          width="27"
+          height="26"
+          viewBox="0 0 27 26"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            className={styles.icon}
+            d="M25 14.75C25.9665 14.75 26.75 13.9665 26.75 13C26.75 12.0335 25.9665 11.25 25 11.25V14.75ZM0.762564 11.7626C0.0791461 12.446 0.0791461 13.554 0.762564 14.2374L11.8995 25.3744C12.5829 26.0578 13.691 26.0578 14.3744 25.3744C15.0578 24.691 15.0578 23.5829 14.3744 22.8995L4.47487 13L14.3744 3.10051C15.0578 2.41709 15.0578 1.30905 14.3744 0.625631C13.691 -0.0577862 12.5829 -0.0577862 11.8995 0.625631L0.762564 11.7626ZM25 11.25L2 11.25V14.75L25 14.75V11.25Z"
+          />
+        </svg>
```

Comment:
You could make the SVGs stand alone files and implement these files here. That would make the code even more readable.

Reply:
Thanks for reviewing! It's on my to Do list for later 👍